require-file-change/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Require file change
+description: Fail unless a specified file is changed by a PR
+
+inputs:
+  override-label:
+    description: Skip check if the PR has this label
+    required: false
+  path:
+    description: Path to file
+    required: true
+  token:
+    description: Personal access token
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Check for ${{ inputs.path }}
+      shell: bash
+      run: |
+        set -euo pipefail
+        if [ -n "${{ inputs.override-label }}" ]; then
+            label=$(echo '${{ toJSON(github.event.pull_request.labels) }}' |
+                    jq '.[] | select(.name == "${{ inputs.override-label }}")')
+            if [ -n "${label}" ]; then
+                echo "PR has ${{ inputs.override-label }} label; skipping"
+                exit 0
+            fi
+        fi
+        diffinfo=$(curl --no-progress-meter \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: token ${{ inputs.token || github.token }}" \
+            "https://api.github.com/repos/${{ github.repository }}/compare/${{ github.event.pull_request.base.sha }}...${{ github.event.pull_request.head.sha }}" |
+            jq '.files[] | select(.filename == "${{ inputs.path }}")')
+        if [ -z "${diffinfo}" ]; then
+            echo "Found no changes to ${{ inputs.path }}."
+            echo "To ignore, add ${{ inputs.override-label }} label to PR."
```

Comment:
Shouldn't this line be conditional on `[ -n "${{ inputs.override-label }}" ]`?

Reply:
Yes, good call.

Reply:
https://github.com/coreos/actions-lib/pull/4